sos/report/plugins/openstack_nova.py

```diff
@@ -137,6 +143,10 @@ def apply_regex_sub(self, regexp, subst):
             self.var_puppet_gen + "_placement/etc/nova/*",
             regexp, subst
         )
+        self.do_path_regex_sub(
+            self.var_puppet_gen + "_metadata/etc/nova/*",
+            regexp, subst
+        )
```

Comment:
Ditto here, does not it make sense to iterate over a list `['', '_libvirt', '_placement', '_metadata']`?

Reply:
done